test/persistence-hooks.suite.js

```diff
@@ -1,7 +1,10 @@
 var ValidationError = require('../').ValidationError;
 var traverse = require('traverse');
 
-module.exports = function(dataSource, should) {
+module.exports = function(dataSource, should, connectorCapabilities) {
+  if (!connectorCapabilities || connectorCapabilities.replaceOrCreateReportsNewInstance === undefined) {
+    console.warn('The connector does not support a recently added feature: isNewInstance reported by replaceOrCreate()');
```

Comment:
On the second thought, perhaps it's better to report the exact capability name here?

```js
console.warn('The connector does not support a recently added feature: replaceOrCreateReportsNewInstance');
```